napi/src/traits.rs

```diff
@@ -84,11 +104,29 @@ impl IntoJsContextual<Vec<ClassInstance<Program>>> for Vec<NodePtr> {
     }
 }
 
+impl IntoJsContextual<Option<ClassInstance<Program>>> for Option<Memos<NodePtr>> {
+    fn into_js_contextual(
+        self,
+        env: Env,
+        this: Reference<ClvmAllocator>,
+        clvm_allocator: &mut ClvmAllocator,
+    ) -> Result<Option<ClassInstance<Program>>> {
+        let Some(memos) = self else {
+            return Ok(None);
+        };
+
+        let ptr = clvm_allocator.0.alloc(&memos.value).map_err(js_err)?;
```

Comment:
is the javascript error not its own error type? I would have expected implementing `From<T>` to make this conversion be implicit by `?`.

Reply:
`napi::Error` and `chia_wallet_sdk::DriverError` are both foreign types so I can't impl `From` due to the orphan rule

Reply:
I see